app/src/css.ts

```diff
@@ -256,6 +256,27 @@ function input() {
   return [base, dark, invertCalendarIcon]
 }
 
+function link() {
+  const base = new Style(':is(.a, a)')
+  base.add('cursor', 'pointer')
+  base.add('text-decoration', 'underline')
+
+  const createStyle = (mode: Mode = 'light') => {
+    const style = new Style(`:is(.a, a)`, mode);
+    style.add('color', mode === 'light' ? l('link.defaultColor') : d('link.defaultColor'));
+    style.add('font-weight', mode === 'light' ? l('link.fontWeight') : d('link.fontWeight'));
+    return style;
+  };
```

Comment:
I understand you're trying to DRY here... but this is creating 3 style blocks for `a` in the output

One with the cursor and text-decoration [static] values and another with color and font-weight. However, we'll want the output to look like:

```css
a {
  cursor: pointer;
  text-decoration: underline;
  color: red;
  font-weight: medium;
}
```